generated/nidaqmx/constants.py

```diff
@@ -925,12 +949,12 @@ class UnitsPreScaled(Enum):
     INCHES_PER_SECOND_SQUARED = 12471  #: Inches per second per second.
     METERS_PER_SECOND = 15959  #: Meters per second.
     INCHES_PER_SECOND = 15960  #: Inches per second.
-    PA = 10081  #: Pascals.
+    PASCALS = 10081  #: Pascals.
     NEWTONS = 15875  #: Newtons.
     POUNDS = 15876  #: Pounds.
     KILOGRAM_FORCE = 15877  #: Kilograms-force.
+    POUNDS_PER_SQUARE_INCH = 15879  #: Pounds per square inch.
```

Comment:
I like this

Reply:
`POUNDS_PER_SQ_INCH` is used in nidaqm-python repo. Thus have retained the enum value name